### PR TITLE
fix(Remindme): removed NLP and use Klasa's argument parser instead

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/commands/Social/remindme.ts [hunk 1]
```diff
@@ -68,46 +68,48 @@ export default class extends SkyraCommand {
...
 
-		let time: number;
-		let attempts = 0;
-		do {
-			const messages = await message.channel.awaitMessages(msg => msg.author === message.author, { time: 30000, max: 1 });
-			if (!messages.size) throw null;
-			time = new Duration(messages.first()!.content).offset;
-			attempts++;
-		} while (time < 60000 && attempts < 5);
+		this.createCustomResolver('idOrDuration', (arg, possible, message, [action]: Actions[]) => {
```
Comment (vladfrangu) [inline src/commands/Social/remindme.ts:94]: The type for the action destructuring should be `[Actions]`

File: src/commands/Social/remindme.ts [hunk 2]
```diff
@@ -68,46 +68,48 @@ export default class extends SkyraCommand {
...
+				case Actions.Create: {
+					if (!arg) throw message.language.tget('COMMAND_REMINDME_CREATE_NO_DURATION');
+					return this.client.arguments.get('timespan')!.run(arg, { ...possible, min: Time.Minute }, message);
+				}
+			}
+		});
 
-		if (!time || time < 60000) throw message.language.tget('COMMAND_REMINDME_SHORT_TIME');
-		return time;
+		this.createCustomResolver('description', (arg, possible, message, [action]: Actions[]) => {
```
Comment (vladfrangu) [inline src/commands/Social/remindme.ts:108]: Ditto

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/commands/Social/remindme.ts (hunk 1 above)
Comment (kyranet) [reply src/commands/Social/remindme.ts:94]: Not possible, it complains that it doesn't match `any[]`.

File: src/commands/Social/remindme.ts (hunk 2 above)
Comment (kyranet) [reply src/commands/Social/remindme.ts:108]: Ditto

**Review by favna — COMMENTED** [review]
> can you also add a show action to reminders to show the full reminders text?

**Comment by kyranet** [comment]
> > can you also add a show action to reminders to show the full reminders text?

Done in d453c77.

File: src/commands/Social/remindme.ts [hunk 3]
```diff
@@ -68,46 +68,50 @@ export default class extends SkyraCommand {
...
-
-			if (indexOfTime !== -1) {
-				parsed.time = new Duration(input.slice(indexOfTime + 4)).offset;
+	public async init() {
+		this.createCustomResolver('action', (arg, _possible, message) => {
+			if (!arg) return Actions.List;
+
+			switch (arg.toLowerCase()) {
+				case 's':
+				case 'show':
```
Comment (favna) [inline src/commands/Social/remindme.ts]:
```suggestion
```

**Review by favna — CHANGES_REQUESTED** [review]

**Review by favna — APPROVED** [review]

**Pull request merged**